components/Search/SearchSuggestion/SearchSuggestion.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { SearchSuggestionItem } from '../SearchSuggestionItem/SearchSuggestionItem'
+import styles from './SearchSuggestion.module.css'
+
+export function SearchSuggestion({ suggestions }) {
+    if (suggestions.length === 0) {
```

Comment:
Let's make the components less smart.

Let the parent component check whether the child component should be render.